dowhy/causal_refuters/bootstrap_refuter.py

```diff
@@ -6,10 +6,10 @@
 class BootstrapRefuter(CausalRefuter):
     """
     Refute an estimate by running it on a random sample of the original data.
-    It supports additional parameters that can be specified in the regute_estimate() method.
-    - 'number_of_samples': int, None by default
-    The number of bootstrap samples to be constructed
-    - 'sample_size': int, None by default
+    It supports additional parameters that can be specified in the refute_estimate() method.
+    - 'num_of_simulations': int, 200 by default

```

Comment:
Thanks @Tanmay-Kulkarni101 for making all these changes. The code looks ready to merge. 
Just one small request before we can merge: to be consistent with causal_estimator, can you shorten "num_of_simulations" to simply "num_simulations" everywhere?

Reply:
Changed num_of_simulations to num_simulations